ehcache/test/play/cache/EhCacheIntegrationTest.java

```diff
@@ -0,0 +1,18 @@
+package play.cache;
+
+import org.junit.jupiter.api.Test;
+import play.Play;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EhCacheIntegrationTest {
+
+  @Test
+  public void initializationWithinRePlay() {
+    Play.configuration.setProperty("ehcache.heapSizeInEntries", "100");
+    Cache.init();
+    Cache.set("no-dummy", 1, "10s");
```

Comment:
@cies  Shouldn't we reset the cache after this test?
Otherwise, this test can affect the following tests.